test/utils/constants.ts

```diff
@@ -0,0 +1,37 @@
+// Extracted from @violetprotocol/extendable/test/utils.js
```

Comment:
Yes it should, and its on the roadmap, but will require a new Extendable update that repackages the npm module and allows importing from there, which currently isn't the case (there are 2 modules, one for contracts, one for the rest of the repo).

This will be included at another later improvement of the package.